### PR TITLE
DOCS-321 - Alt text for security articles

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request adds and updates alt text for these articles:
* https://help.sumologic.com/docs/security/
* https://help.sumologic.com/docs/manage/security/
* https://help.sumologic.com/docs/manage/users-roles/

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/cse/administration/index.md [hunk 1]
```diff
@@ -12,103 +12,103 @@ Learn about onboarding tasks and best practices for Cloud SIEM administrators. I
...
   <div className="container">
-  <a href="/docs/cse/administration/save-inventory-data-lookup-table"><img src={useBaseUrl('img/icons/operations/frequent-search.png')} alt="icon" width="40"/><h4>Save Inventory Data to a Lookup Table</h4></a>
+  <a href="/docs/cse/administration/save-inventory-data-lookup-table"><img src={useBaseUrl('img/icons/operations/frequent-search.png')} alt="Check mark in an magnifhing glass icon" width="40"/><h4>Save Inventory Data to a Lookup Table</h4></a>
   <p>Learn how to use a saved Sumo Logic search to populate a Lookup Table with Cloud SIEM inventory data.</p>
   </div>
 </div>
 <div className="box smallbox card">
   <div className="container">
-  <a href="/docs/cse/administration/manage-custom-insight-resolutions"><img src={useBaseUrl('img/icons/operations/correlation-engine.png')} alt="icon" width="40"/><h4>Manage Custom Insight Resolutions</h4></a>
+  <a href="/docs/cse/administration/manage-custom-insight-resolutions"><img src={useBaseUrl('img/icons/operations/correlation-engine.png')} alt="Barchart with a gear icon" width="40"/><h4>Manage Custom Insight Resolutions</h4></a>
```
Comment (kimsauce) [inline docs/cse/administration/index.md:87]:
```suggestion
  <a href="/docs/cse/administration/manage-custom-insight-resolutions"><img src={useBaseUrl('img/icons/operations/correlation-engine.png')} alt="Bar chart with a gear icon" width="40"/><h4>Manage Custom Insight Resolutions</h4></a>
```

File: docs/cse/administration/index.md [hunk 2]
```diff
@@ -12,103 +12,103 @@ Learn about onboarding tasks and best practices for Cloud SIEM administrators. I
...
   <p>Learn how to create and manage Custom Insight Resolutions.</p>
   </div>
 </div>
 <div className="box smallbox card">
   <div className="container">
-  <a href="/docs/cse/administration/manage-custom-insight-statuses"><img src={useBaseUrl('img/icons/operations/retain-and-visualize-logs.png')} alt="icon" width="40"/><h4>Managing Custom Insight Statuses</h4></a>
+  <a href="/docs/cse/administration/manage-custom-insight-statuses"><img src={useBaseUrl('img/icons/operations/retain-and-visualize-logs.png')} alt="Barchart in a circle icon" width="40"/><h4>Managing Custom Insight Statuses</h4></a>
```
Comment (kimsauce) [inline docs/cse/administration/index.md:93]:
```suggestion
  <a href="/docs/cse/administration/manage-custom-insight-statuses"><img src={useBaseUrl('img/icons/operations/retain-and-visualize-logs.png')} alt="Bar chart in a circle icon" width="40"/><h4>Managing Custom Insight Statuses</h4></a>
```

File: docs/cse/integrations/enable-virustotal-enrichment.md [hunk 3]
```diff
@@ -37,8 +37,8 @@ VirusTotal enrichments are only added to Signals that are part of an Insight.
...
-1. On the **Enrichment** page, click the pencil icon for VirusTotal.<br/><img src={useBaseUrl('img/cse/enrichment-page.png')} alt="enrichment-page.png" style={{border: '1px solid gray'}} width="800" />
-2. On the **Edit VirusTotal Configuration** popup, enter your VirusTotal API Key, and click Update.<br/><img src={useBaseUrl('img/cse/edit.png')} alt="enrichment-page.png" style={{border: '1px solid gray'}} width="400" />
+1. On the **Enrichment** page, click the pencil icon for VirusTotal.<br/><img src={useBaseUrl('img/cse/enrichment-page.png')} alt="Edit button on the VirusTotal enrichment" style={{border: '1px solid gray'}} width="800" />
+2. On the **Edit VirusTotal Configuration** popup, enter your VirusTotal API Key, and click Update.<br/><img src={useBaseUrl('img/cse/edit.png')} alt="Edit VirusTotal Configuration popup" style={{border: '1px solid gray'}} width="400" />
```
Comment (kimsauce) [inline docs/cse/integrations/enable-virustotal-enrichment.md:41]: Per [MS Style Guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/pop-up)

```suggestion
2. On the **Edit VirusTotal Configuration** popup, enter your VirusTotal API Key, and click Update.<br/><img src={useBaseUrl('img/cse/edit.png')} alt="Edit VirusTotal Configuration pop-up" style={{border: '1px solid gray'}} width="400" />
```

**Review by kimsauce — APPROVED** [review]
Awesome job, John! Thanks for your hard work & consistency on this.

**Comment by jpipkin1** [comment]
Thanks for the review, @kimsauce!

**Pull request merged**